migrations/20200330130157_add_review_date_to_pils.js

```diff
@@ -0,0 +1,23 @@
+const moment = require('moment');
+
+exports.up = function(knex) {
+  return knex
+    .select('id', 'updated_at', 'review_date')
+    .from('pils')
+    .whereNull('review_date')
+    .then(pils => {
+      return pils.reduce((promise, pil) => {
+        const { id, updated_at } = pil;
+        const review_date = moment(updated_at).add(5, 'years').toISOString();
+        return promise.then(() => {
+          return knex('pils')
+            .where({ id })
+            .update({ review_date })
+        })
+      }, Promise.resolve());
+    });
+};
+
+exports.down = function(knex) {
+
```

Comment:
```suggestion
  return Promise.resolve();
```

Not sure it makes a difference, but my last migration beefed about not returning a promise in the rollback.